### DISCUSSION TITLE
Safety mode triggered by unchanging outdoor temp?

### DISCUSSION THREAD

**Discussion by R5fan** [jmcollin78/versatile_thermostat#746] [General]
I keep getting unsafe security mode, despite my temp sensor working  fine and having the safety delay set to 120 mins. It looks like an unchanging outdoor temperature triggers this, which makes no sense:

<img width="410" alt="image" src="https://github.com/user-attachments/assets/ea1eaf43-7369-4544-bfd3-266a68015335" />

**Comment by jmcollin78** [comment]
Hello, you can deactivate the safety mode for outdoor temperature. See Readme.

**Reply by R5fan** [reply]
Indeed, I missed that, thanks. Still doesnt make much sense to me its enabled by default (and I think at 60min) and requires YAML to disable.

**Reply by jmcollin78** [reply]
External temperature is used by the TPI algo. So if it is not checked, you could have wrong setpoint in your house.

**Reply by R5fan** [reply]
The issue is that outside temp may not change for many hours and many sensors dont send updates if there is no change. Here is the forecast for my area now:
<img width="865" alt="image" src="https://github.com/user-attachments/assets/762ba476-41c3-4beb-ae11-5dbca3781332" />

Even if the sensor does go offline, its not a safety risk, at worst you may get sub optimal regulation (but the effect will be extremely minimal in almost all cases)

**Reply by jmcollin78** [reply]
That is why you can deactivate the check of the outdoor temp sensor.
